Controller/CRUDController.php

```diff
@@ -221,7 +221,8 @@ public function deleteAction($id)
     /**
      * Edit action.
      *
-     * @param int|string|null $id Optional ID to use instead of globally provided default.
+     * @param int|string|null $id Optional ID representing the main admin subject.
```

Comment:
The `$id` parameter was introduced in 3d0cc1f8e7daf3e9c8f4c1945d94b8ed8dc67935 , and was used!